packages/components/psammead-figure/package-lock.json

```diff
@@ -35,6 +35,49 @@
       "resolved": "https://registry.npmjs.org/@bbc/gel-foundations/-/gel-foundations-0.1.4.tgz",
```

Comment:
You'll want to also increment the version in the `package-lock.json` & add a row in the `CHANGELOG.md` for both components.